### PR TITLE
Added Subscription UI, Changed Bootstrap Version to 3, Modified Protr…

### PR DESCRIPTION
**Pull request by saiprasad1997**
…actor configuration

### CODE REVIEW

File: src/app/subscription/subscription.component.html [hunk 1]
```diff
@@ -1,3 +1,110 @@
...
+
+
+        <div class="row">
+          <div class="field col-sm-6">
+            <div class="left-textbox pull-left">
+              <label class="lbl-size">FLOW TYPE</label>
+              <span class="required-star">*</span>
+              <div class="item">
+                <div class="b-dropdown dropdown">
+                <button class="btn dropdown-toggle big" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
```
Comment (pooja-koul-incontact) [inline src/app/subscription/subscription.component.html]: Give proper names to IDs and other identifiers.

File: src/app/subscription/subscription.component.ts [hunk 2]
```diff
@@ -1,15 +1,89 @@
...
+    return this.mySwitch;
+  }
+
+  onSubmit() {
+    this.submitted = true;
+
+    if (this.subscriptionForm.invalid) {
+      return;
+    }
+    const e = document.getElementById('default-text').innerText;
```
Comment (pooja-koul-incontact) [inline src/app/subscription/subscription.component.ts]: What is e?

File: src/app/subscription/subscription.component.css [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+
+.create-subscription-data .b-switch {
+  position: relative;
+  display: inline-block;
+  font-family: "OpenSans";
+  color:#fff;
+  font-size:12px;
+}
+
+/* Hide default HTML checkbox */
```
Comment (pooja-koul-incontact) [inline src/app/subscription/subscription.component.css]: Are these comments required?

File: src/app/subscription/subscription.component.ts (hunk 2 above)
Comment (saiprasad1997) [reply src/app/subscription/subscription.component.ts]: Need to change variable name

File: src/app/subscription/subscription.component.css (hunk 3 above)
Comment (saiprasad1997) [reply src/app/subscription/subscription.component.css]: Yes. As this component is taken from Standard UX Controls , we should know the purpose of each Class and ID

File: src/app/subscription/subscription.component.ts (hunk 2 above)
Comment (abhilash-incontact) [reply src/app/subscription/subscription.component.ts]: For constants use capital case. eg const PI = 3.146

**Pull request merged**